src/scss/components/button.scss

```diff
@@ -0,0 +1,16 @@
+.btn-secondary {
+  border-color: $gray-300;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
```

Comment:
❓ Why don't we have left radius on `btn-secondary`?

Reply:
I assume you mean `right-radius`?

Reply:
That doesn't make sense, maybe it was done for the search button attached to the field?

Reply:
> I assume you mean right-radius?

It's removing the left radius - and only for `btn-secondary`. All the other button variants have radius on both sides

> maybe it was done for the search button attached to the field

Possibly. The template should be updated to use the right DOM structure if that's the case. Bootstrap 4 has documentation for doing this.

Reply:
Oh sorry yeah, it's setting only left (what I saw) but _to 0_ (what you saw) 😆 - thanks :)
Agree with statement - bootstrap has patterns for putting buttons of the end of inputs.